support-frontend/assets/helpers/fontLoader.js

```diff
@@ -1,17 +1,50 @@
 // @flow
+import * as storage from 'helpers/storage';
+
 
 const fetchFonts = (window: Object, document: Document): void => {
   const head: null | HTMLHeadElement = document.querySelector('head');
 
-  const useFont = (font: { css: string }): void => {
-    if (head) {
+  const useFont = (font: { fontName: string, css: string }): void => {
+    const fontClassName = `gu_font__${font.fontName}`;
+    if (document.getElementsByClassName(fontClassName).length === 0 && head && font.fontName && font.css) {
       const style: HTMLStyleElement = document.createElement('style');
+      style.classList.add(`gu_font__${font.fontName}`);
       style.innerHTML = font.css;
       head.appendChild(style);
     }
   };
 
+  const saveFontToLocalStorage = (font: { fontName: string, css: string }) => {
+    const { fontName } = font;
+    if (fontName) {
+      const currentFontsString = storage.getLocal('guFonts');
+      const currentFonts = currentFontsString ? JSON.parse(currentFontsString) : {};
+      storage.setLocal('guFonts', JSON.stringify({
```

Comment:
@jranks123 should we add a check around here to see if the typeface is already in localStorage.guFonts? Not sure if writing a second typeface with the same name will replace the previous typeface or if it will create a new one. If it creates a new typeface, we'll be incrementing the time to run loadFontsFromLocalStorage every time a user navigates around the site, I think?

Reply:
This will overwrite what is there. 

In case you haven't come across it before, this bit of code uses [spread syntax ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals)(the `...`) :"

```
storage.setLocal('guFonts', JSON.stringify({
        ...currentFonts,
        [fontName]: font,
      }));
```
So the result of 

```{
        ...currentFonts,
        [fontName]: font,
      }
```
will be whatever currentFonts is, with the value for [fontName] either added if it isn't already in the object, or with the current value replaced with the value of `font`. 

And then `storage.setLocal` will overwrite whatever is there. 


And I think that is preferable, as it is possible that the fonts from dotcom may change, so we do want to overwrite it each time, to be safe, I think

Reply:
Ok cool, just wanted to make sure there wasn't duplicating. Nice work.